tests/stream/test_stream_smoke/0022_udf2.yaml

```diff
@@ -0,0 +1,174 @@
+test_suite_name: udf2
+tag: smoke
+test_suite_config:
+  tests_2_run:
+    ids_2_run:
+      - all
+    tags_2_run: [ ]
+    tags_2_skip:
+      default:
+        - todo
+        - to_support
+        - change
+        - bug
+        - sample
+      cluster:
+        - view
+        - cluster_table_bug
+comments: Test SQL to create UDF/UDA.
+
+tests:
+  - id: 21
+    tags:
+      - udf2
+    name: SQL create scalar UDF
+    description: SQL - scalar UDF
+    steps:
+      - statements:
+          - client: python
+            query_type: table
+            wait: 1
+            query: DROP FUNCTION IF EXISTS test_add_five_5;
+
+          - client: python
+            query_type: table
+            query_id: udf-21-0
+            wait: 1
+            query: |
+              CREATE OR REPLACE FUNCTION test_add_five_5(value float32) RETURNS float32 LANGUAGE JAVASCRIPT AS $$
+                function test_add_five_5(value) {
+                  for(let i=0;i<value.length;i++) {
+                    value[i]=value[i]+5;
+                  }
+                  return value;
+                }
+              $$;
+
+          - client: python
+            query_id: udf-21-1
+            query_end_timer: 7
+            depends_on_done: udf-21-0
+            query_type: table
+            wait: 5
+            query: |
+              DESCRIBE (SELECT test_add_five_5(1) AS a);
+
+          - client: python
+            query_type: table
+            query_id: udf-21-2
+            depends_on_done: udf-21-1
+            wait: 1
+            query: |
+              DROP FUNCTION test_add_five_5;
+
+          - client: python
+            query_id: udf-21-3
+            query_end_timer: 1
+            query_type: table
+            depends_on_done: udf-21-2
+            wait: 5
+            query: |
+              SELECT test_add_five_5(1)
+
+    expected_results:
+      - query_id: udf-21-1
+        expected_results:
+          - [ 'a', 'float32', '', '', '', '', '' ] # DESCRIBE (SELECT test_add_five_5(1) AS a)
+      - query_id: udf-21-3
+        expected_results: 'error_code:46' # SELECT test_add_five_5(1) after DROP FUNCTION
+
+  - id: 22
+    tags:
+      - udf2
+    name: SQL create UDA
+    description: SQL - UDA
```

Comment:
Can we create a test case for UDA with customized emit strategy ?

Reply:
It will be added in another PR next to this one.